src/main/java/ch/njol/unofficialmonumentamod/features/locations/Locations.java

```diff
@@ -73,6 +75,15 @@ public static String getShardFrom(Text text) {
 		return shard;
 	}
 
+	public static String getShortShardFrom(Text text) {
+		String fullShard = getShardFrom(text);
+		if (fullShard == null) {
+			return null;
+		}
+
+		return cachedShard.replaceFirst("-\\d+$", "");
```

Comment:
Shouldn't this be `fullShard.replaceFirst`? It currently mostly ignores the argument.

Reply:
ye, not supposed to be calling the cache.